spec/index.html

```diff
@@ -866,8 +866,10 @@ <h2>Triple-term based reification</h2>
 
   <section id="ch_proposition">
     <h3><code>rdfs:Proposition</code></h3>
-    <p><code>rdfs:Proposition</code> is the class of reified triples.
-      <code>rdfs:Proposition</code> is an instance of <a href="#ch_class"><code>rdfs:Class</code></a>.</p>
+    <p><code>rdfs:Proposition</code> is the class of abstract, atomic, logical propositions.
```

Comment:
As above, I would prefer to reuse the [definition given in RDF-Concepts](https://www.w3.org/TR/rdf12-concepts/#dfn-proposition), which I find clearer.

```suggestion
    <p><code>rdfs:Proposition</code> is the class of propositions, simple logical expressions, describing a relationship between two entities.
```

Possibly we could even add more details

```suggestion
    <p><code>rdfs:Proposition</code> is the class of propositions, simple logical expressions, describing a relationship between two entities, as denoted (in particular) by a triple term.
```